### PR TITLE
Add Carousel Card for Projects

### PR DESCRIPTION
**Pull request by DuongTyler**

### CODE REVIEW

File: frontend/src/components/gallery/artworkCard/artworkCard.css [hunk 1]
```diff
@@ -10,15 +10,17 @@
...
     position: relative;
     transition: all 0.4s ease;
+    object-fit: contain;
+    background-color: #000;
 }
 
 .artwork-card-img > * {
     width: 100%;
-    /*height: auto;*/
+    height: 100%;
```
Comment (zapplecat) [inline frontend/src/components/gallery/artworkCard/artworkCard.css]: Do we really want 100% for both of this...?

Comment (zapplecat) [reply frontend/src/components/gallery/artworkCard/artworkCard.css]: I think this 100% might be the cause of this on smaller screens 
image

File: frontend/src/pages/home/home.tsx [hunk 2]
```diff
@@ -363,13 +371,14 @@ Thank you."}
                             </FadeIn>
                             <div className="project-separator" />
                             <FadeIn className="fade-in">
-                                <ProjectCard
+                                <MultiProjectCard
```
Comment (zapplecat) [inline frontend/src/pages/home/home.tsx:374]: Probably shouldn't be this PR, you could potentially consider using modal to zoom into the carousel like the ImageCard.

File: frontend/src/components/projectSection/multiProjectCard.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+interface MultiProjectCardProps {
+    username: string,
+    title: string,
+    description: string,
+    images: string[],
+    projectlink: string,
+}
+
+export default class MultiProjectCard extends Component<MultiProjectCardProps>{
```
Comment (zapplecat) [inline frontend/src/components/projectSection/multiProjectCard.tsx:15]: Just a thought, why don't we combine the ProjectCard into here. And for anything with `this.props.images < 2` we render using the normal ProjectCard method, and everything else we can use the carousel.

EDIT: Maybe `this.props.images === 1` actually.

File: frontend/src/components/gallery/artworkCard/artworkCard.css (hunk 1 above)
Comment (zapplecat) [reply frontend/src/components/gallery/artworkCard/artworkCard.css]: nvm, I can't reproduce it anymore. Might be a resize thing.

**Review by zapplecat — APPROVED** [review]
Left some comments about various things that don't need to be addressed as a part of this PR.

File: frontend/src/components/projectSection/multiProjectCard.tsx (hunk 3 above)
Comment (DuongTyler) [reply frontend/src/components/projectSection/multiProjectCard.tsx:15]: that's a good idea. I'll try to do that tomorrow :p

File: frontend/src/components/gallery/artworkCard/artworkCard.css (hunk 1 above)
Comment (DuongTyler) [reply frontend/src/components/gallery/artworkCard/artworkCard.css]: ah yea, this actually had no effect on the result of the carousel. I'll just remove it.

**Pull request merged**